plugins/dynatrace/src/components/Synthetics/SyntheticsCard/SyntheticsCard.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import useAsync from 'react-use/lib/useAsync';
+import { Progress } from '@backstage/core-components';
+import Alert from '@material-ui/lab/Alert';
+import { InfoCard } from '@backstage/core-components';
+import { useApi } from '@backstage/core-plugin-api';
+import { dynatraceApiRef } from '../../../api';
+import { SyntheticsLocation } from '../SyntheticsLocation';
+
+type SyntheticsCardProps = {
+  syntheticsId: string;
+  dynatraceBaseUrl: string;
+};
+
+const dynatraceMonitorPrefixes = (idPrefix: string): string => {
+  switch (idPrefix) {
+    case 'HTTP_CHECK':
+      return 'ui/http-monitor';
+    case 'BROWSER_MONITOR':
+      return 'ui/browser-monitor';
+    case 'SYNTHETIC_TEST':
+      return 'ui/browser-monitor';
+    default:
+      throw new Error('Invalid synthetic Id');
+  }
+};
+
+export const SyntheticsCard = (props: SyntheticsCardProps) => {
+  const { syntheticsId, dynatraceBaseUrl } = props;
+  const dynatraceApi = useApi(dynatraceApiRef);
+  const { value, loading, error } = useAsync(async () => {
+    return dynatraceApi.getDynatraceSyntheticFailures(syntheticsId);
+  }, [dynatraceApi, syntheticsId]);
+
+  if (loading) {
+    return <Progress />;
+  } else if (error) {
+    return <Alert severity="error">{error.message}</Alert>;
+  }
+
+  const deepLinkPrefix = dynatraceMonitorPrefixes(
+    `${syntheticsId.match(/(.+)-/)![1]}`,
```

Comment:
`syntheticsId.split('-')[0]` may be a bit clearer, i had to read this twice to figure it out :) plus it will fail if there's no dash